client/src/research/LoginPage.js

```diff
@@ -1,11 +1,60 @@
 import React, { Component } from 'react';
+import './LoginPage.css';
 
 
 // The page for users to login for accessing research data.
 class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
This is great.  As a convention, please use `onSomethingHappened` as the format for functions that handle a particular event.  The linter rules will also check that those come before "render" methods in components.

Reply:
gotcha! I've changed my naming convention :)